frontend/src/js/controllers/teamsCtrl.js

```diff
@@ -359,6 +359,55 @@
             }, function() {
             });
         };
+
+
+        vm.editTeamName = function(ev, participantTeamId) {
+            ev.stopPropagation();
+            // Appending dialog to document.body to cover sidenav in docs app
```

Comment:
Can you please modify the comment to make it meaningful ? Also please add comments where ever you think they are needed.